app-portage/conf-update/conf-update-1.0.3-r1.ebuild

```diff
@@ -14,18 +14,22 @@ SLOT="0"
 KEYWORDS="amd64 ppc x86"
 IUSE="colordiff libressl"
 
-RDEPEND=">=dev-libs/glib-2.6
-		sys-libs/ncurses:0
-		!libressl? ( dev-libs/openssl:0 )
-		libressl? ( dev-libs/libressl )
-		colordiff? ( app-misc/colordiff )"
-DEPEND="virtual/pkgconfig
-		${RDEPEND}"
+RDEPEND="
+	dev-libs/glib
+	sys-libs/ncurses:0
+	colordiff? ( app-misc/colordiff )
+	!libressl? ( dev-libs/openssl:0 )
+	libressl? ( dev-libs/libressl )"
```

Comment:
both libressl and openssl require `:0=` when linking (please check)

Reply:
Added